_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1046 @@
+> > The first step here is to check if the variable "cVegStderr" is already
+> > listed in a CMOR table. We have the information that the modeling_realm
+> > of the variable is ``land`` and that the frequency is ``mon``. This
+> > means we have to check the CMOR table "Lmon" for an entry. We focus
+> > our search on the CMIP6 tables since these are the most recent tables
+> > available. You can find the lists here:
+> > `<https://github.com/ESMValGroup/ESMValCore/tree/master/esmvalcore/cmor/tables/cmip6/Tables>`
+> >
+> > We do not find the variable "cVegStderr" in the "Lmon" table, which
+> > means we will have to write our own custom table for this.
+> > There were two examples given which we could use as templates for the
+> > new custom table that we have to create. These two examples can be
+> > found here: [cVeg](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Lmon.json)
+> > and [gppStderr](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/custom/CMOR_gppStderr.dat)
+> >
+> > We have to create a new file with the name ``CMOR_cVegStrerr.dat`` in
+> > the custom CMOR table folder (https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/custom/).
+> > The content of the file should then look like this:
+> >
+> > ```
+> > SOURCE: CMIP6
+> > !============
+> > variable_entry:    cVegStderr
+> > !============
+> > modeling_realm:    land
+> > !----------------------------------
+> > ! Variable attributes:
+> > !----------------------------------
+> > standard_name:
+> > units:             kg m-2
+> > cell_methods:      area: mean where land time: mean
+> > cell_measures:     area: areacella
+> > long_name:         Carbon Mass in Vegetation Error
+> > !----------------------------------
+> > ! Additional variable information:
+> > !----------------------------------
+> > dimensions:        longitude latitude time
+> > out_name:          cVegStderr
+> > type:              real
+> > !----------------------------------
+> > !
+> > ```
+> >
+> > Note that there is no entry for ``standard_name``. This is on purpose.
+> > It is a sign for the ESMValTool to not crash although the variable that
+> > we are looking for is ok to have no official CMIP6 ``standard_name``.
+> >
+> {: .solution}
+{: .challenge}
+
+## 5. Create a CMORizer for the new dataset
+
+Now that we have the data ready, have told the ESMValTool where to look
+for it, and have made sure that our variable of choice is listed either
+on a pre-existing or custom CMOR table, let's test if the data is actually 
+following the necessary CMOR standard already, or if we have to do some
+reformatting for the dataset.
+
+> ## Run the test recipe again
+>
+> Run the test recipe again from earlier where you wanted to check if the
+> ESMValTool can already find and read the freshly downloaded FLUXCOM data.
+>
+> > ## Answers
+> >
+> > ```bash
+> > esmvaltool run recipe_check_fluxnet.yml --log_level debug
```

Comment:
I think it would still not find the data at this point since we have not yet run the cmorizer, right? So all we have is RAWOBS, whereas we need some data in OBS6 to get this error.

Maybe we still need to move the section "## Check if your variable is following the CMOR standard / Check if it's in a CMOR table" further down, so that the first thing we do is run the (almost) empty cmorizer script.

Reply:
@Peter9192: yes, you are right. The ESMValTool would not find the data yet. It just did for me, I remember now, because I had it look in the RAWOBS folder...
And yes, I agree, we probably should have the "CMOR standard section" a little further down. Or at least have the people try to run an almost empty cmorizer script after the section "Location, location, location".

Reply:
@Peter9192: actually, I have thought about a different approach, how to adjust our story line so that it makes more sense. I have implemented that now. Let me know what you think!